src/main/java/datagateways/DestinyCardDataManager.java

```diff
@@ -0,0 +1,44 @@
+package datagateways;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * The DestinyCardDataManager class is responsible for reading destiny card data from a CSV file and providing it as an ArrayList of String arrays.
+ */
+public class DestinyCardDataManager implements DataManager{
+
+    /**
+     * Creates destiny cards based on the information in the specified CSV file.
+     *
+     * @param fileName The name of the CSV file containing destiny card information.
+     * @return An ArrayList of String arrays representing the data read from the CSV file.
+     */
+    public ArrayList<String[]> readData(String fileName){
+
+        ArrayList<String[]> lines = new ArrayList<>();
+
+        try (BufferedReader br = new BufferedReader(new FileReader(fileName))) {
+            String line;
+
+            while ((line = br.readLine()) != null){
+                String[] values = line.split(DELIMITER);
+                if (values.length < 4) {
```

Comment:
Insted of this magic number `4`, do we want to reduce code smell and make a `public final int fieldNum = 4` on top of the file?